integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/inbound-endpoints/synapse-websocket-inbound.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+    <sequence name="dispatchSeq" xmlns="http://ws.apache.org/ns/synapse">
+        <property name="OUT_ONLY" value="true"/>
+        <log level="full">
+            <property name="LOGGED_MESSAGE" value="LOGGED"/>
```

Comment:
Let's add a unique message, something like, "websocket inbound invoked"?

Reply:
Fixed in acbeee0db79361849a67c4ade85a802ee666242f